odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -3,6 +3,12 @@
 """
 
 renamed_modules = {
+    # Odoo
+    'base_action_rule': 'base_automation',
+    'crm_project_issue': 'crm_project',
+    'stock_picking_wave': 'stock_picking_batch',
+    'website_issue': 'website_form_project',
```

Comment:
Are you sure this is renaming? I see it more as merging, specially the last one

Reply:
Same thing as far as the analysis is concerned. I'd like to have all the merged modules in here.

Reply:
So this list is not complete but it's a start.

Reply:
Now, separated in two lists, I think they are complete.